packages/sanity/src/core/store/bundles/types.ts

```diff
@@ -7,6 +8,7 @@ export interface BundleDocument extends SanityDocument {
   name: string
   description?: string
   tone?: ButtonTone
-  icon?: string
+  icon?: IconSymbol
   authorId: string
+  publishedAt?: Date
```

Comment:
Even though we will use it as a date, the actual value returned from the document will be a string in this case.